examples/gridsearch_launch.py

```diff
@@ -0,0 +1,90 @@
+"""
+Neuroimaging cartesian reconstruction
+=====================================
+
+Author: Chaithya G R
+
+In this tutorial we will use the pysap-mri's launch grid helper function
+to carry out grid search. We will search for best regulatization weight
+reconstruction and best wavelet for reconstruction, while search for a
```

Comment:
I think the first reconstruction is not supposed to be here

Reply:
What do you mean by:
> while search for a space of 5 weight values

Reply:
Ahh, that's bad English fixing. What I wanted to say was that we search for best value of `mu` searched in 5 values in a logarithmic grid